### ISSUE TITLE
Samseong Galaxy Tab A6 10.1 - Not processing FLAC

### ISSUE THREAD

**Issue by LokeYourLord** [ShadySquirrel/ViPER4Android#34]
So first I had an issue with that the module didn't work at all, it was "Enabled: Yes" and everything, but "Processing" was always on No. Then I uninstalled it, installed Universal_Deep_Buffer_Remover and then your module, and then it worked, however, it does not work when I try to play FLAC files in Apollo, it does however work on Moto Z Droid just fine (both Android 7.1.1). I use the same Magisk Modules on both my Phone and Tablet (Samseong Galaxy Tab A6 10.1), but on my Tablet (same Music Player as on my Phone, Apollo Material) it does not modify the output at all when playing FLAC, only MP3 or such. How can I fix this?

And to answer a question, I do NOT have any other sound mods BESIDES Universal_Deep_Buffer_Remover installed (because I actually need it, and the fact that the module doesn't work at all without on some Apps, so I know it doesn't cause any harm as it works flawless on my Moto Z Droid). So I wonder what it might be...

Thanks in advance :D

**Comment by ShadySquirrel** [comment]
You can try with different player, or check if there is some setting inside that one, but basically this is more of a codec/effect library/device issue than module issue so I can't help much. All formats are working on devices I'm testing with.

**Comment by LokeYourLord** [comment]
The problem is that I have had this exact same issue before on my old Note 4 a while ago, but I didn't use Magisk and this Module back then, just regular V4A. There is nothing inside that player, I mean why would there be if it works just fine on my Moto Z. Would it be possible for you to inspect some logs and files and tell me if it is supposed to work? That would be great, just tell me what you need :)

**Comment by ShadySquirrel** [comment]
Logs are always a good thing, yes. It's all described in README, I'm from my phone ATM so it's a bit pain to link.

Also, if your firmware has any additional effects it would be nice to know which ones and paths where they are.

**Comment by ShadySquirrel** [comment]
Closed for inactivity.

**Comment by LokeYourLord** [comment]
Sorry about that, was very busy. Anyways, I fixed it, I had to actually switch out the audio_effects.conf with a modified one for Samsung (X, in general) X Phone(s)/Tablet(s). I can upload it here (I also had to use TWRP to force replace the one in the system at /system/etc/audio_effects.conf, Magisk did not want to work). I also actually had to fully remove SoundAlive, not juse use FX Compatible in Viper4Android.

Here the config:
[audio_effects.zip](https://github.com/ShadySquirrel/ViPER4Android/files/1776769/audio_effects.zip)
